ADOPTERS.md

```diff
@@ -0,0 +1,7 @@
+## Hydrophone Adopters
+
+A non-exhaustive list of hydrophone adopters is provided below.
+
+**[Talos Linux](https://www.talos.dev/)** - placeholder
```

Comment:
Talos Linux uses `hydrophone` to run Kubernetes integration tests in several ways:

* `talosctl health --run-e2e` runs a short Kubernetes integration test after running Talos health checks
* `talosctl conformance` allows to run full integration test suite both "fast" mode (non-Serial tests in parallel, and Serial tests sequentially), and "certified" mode producing a bundle suitable for certification purposes

This is used both in continuous integration and available to end-users to verify their cluster configuratiion.